### PR TITLE
ecies: use align_num value

### PR DESCRIPTION
**Pull request by nkysg**

### CODE REVIEW

File: crates/net/ecies/src/algorithm.rs [hunk 1]
```diff
@@ -273,6 +273,12 @@ pub struct RLPxSymmetricKeys {
     pub mac_key: B256,
 }
 
+/// return num alignment value
+#[inline]
+const fn align_num(num: usize, align: usize) -> usize {
+    (num + (align - 1)) & !(align - 1)
+}
```
Comment (mattsse) [inline crates/net/ecies/src/algorithm.rs:280]: this should be moved below the impl block

**Review by mattsse — CHANGES_REQUESTED** [review]

File: crates/net/ecies/src/algorithm.rs [hunk 2]
```diff
@@ -730,6 +730,12 @@ impl ECIES {
         self.ingress_aes.as_mut().unwrap().apply_keystream(ret);
         Ok(split_at_mut(ret, size)?.0)
     }
+
+    /// return num alignment value
+    #[inline]
+    const fn align_num(num: usize, align: usize) -> usize {
+        (num + (align - 1)) & !(align - 1)
+    }
```
Comment (mattsse) [inline crates/net/ecies/src/algorithm.rs:738]: @DaniPopes need help here 🙃, is this euqivalent

Comment (DaniPopes) [reply crates/net/ecies/src/algorithm.rs:738]: this should be equal to "<integer>.next_multiple_of"

File: crates/net/ecies/src/algorithm.rs [hunk 3]
```diff
@@ -730,6 +730,13 @@ impl ECIES {
         self.ingress_aes.as_mut().unwrap().apply_keystream(ret);
         Ok(split_at_mut(ret, size)?.0)
     }
+
+    /// return num alignment value
+    /// `<https://stackoverflow.com/questions/14561402/how-is-this-size-alignment-working>`
```
Comment (DaniPopes) [inline crates/net/ecies/src/algorithm.rs:735]:
```suggestion
    /// Returns `num` aligned to 16.
    ///
    /// `<https://stackoverflow.com/questions/14561402/how-is-this-size-alignment-working>`
```

**Review by DaniPopes — CHANGES_REQUESTED** [review]

**Review by DaniPopes — APPROVED** [review]

**Review by mattsse — APPROVED** [review]

**Pull request merged**